docs/src/pages/premium-themes/onepirate/modules/components/Typography.js

```diff
@@ -102,8 +102,6 @@ Typography.propTypes = {
     'body2',
     'button',
     'caption',
```

Comment:
whatever i did, seems to work, but removed these

Reply:
these got auto-generated for me upon building, so i wonder if even though we manually removed them here they will come back with the next checkin that builds the project; we will see :)